src/main/java/UI.java

```diff
@@ -102,10 +102,11 @@ public void printPatientMenuMessage() {
         System.out.println(PATIENT_MAIN_MENU);
     }
 
-    public void printPatientAddedMessage(Patient patient) {
-        System.out.println("Ok! I've added a patient! The patient's details are as follows:");
+    public void printMessageAndObject(String object, String message, int index) {
+        System.out.println(message);
         printLine();
-        System.out.println(patient);
+        System.out.println((index + 1) + ")");
```

Comment:
I think index should be printed in `Prescription #1` or `Visit #1` format to align with the other viewing functions, you may check the print messages for viewing functions e.g viewAll